roles/edpm_nova/tasks/install.yml

```diff
@@ -1,5 +1,25 @@
 ---
 
+- name: Create a directory for container health checks
+  ansible.builtin.file:
+    path: /var/lib/containers/healthchecks
+    state: directory
+    setype: container_file_t
+    owner: root
+    group: root
+    mode: '0755'
+  become: true
+
+- name: Deploy multipathd health check script
+  ansible.builtin.copy:
+    src: healthchecks/nova_compute/
+    dest: /var/lib/containers/healthchecks/nova_compute
```

Comment:
the file would be createdin /var/lib/openstack/nova

again become:true is not required and it should not be owned by root.

Reply:
ack